examples/tools/system/consuming_system_packages.rst

```diff
@@ -0,0 +1,111 @@
+.. _examples_tools_system_consuming_system_packages:
+
+Consuming system requirements only when building a package
+==========================================================
+
+In some cases, you may want to consume system requirements only when building a package, but not when installing it.
+It can be useful when you want to build a package in a CI/CD pipeline, but you don't want to run the system package
+manager when installing the Conan package in a different environment.
+For those cases, there are few approaches that can be used to achieve this goal.
+
+
+Consume a Conan package wrapper for a system package using hidden visibility
```

Comment:
Isn't this mostly what is discussed in the other example, with the system wrapper? Why repeating it here, not better linking to the other example?

Reply:
That is exactly my point. We have a full working and reproducible example in https://docs.conan.io/2/examples/tools/system/package_manager.html. Why adding a new partial ``conanfile.py`` here, just for the ``visible=True``? I think the best would be to provide the real world case:
- Provide a full ``pkg`` conanfile.
- Then use the ``tool_requires()`` instead of ``visible=False``, it is more idiomatic and adequate for the task.
- ``conan create`` the package
- Consume the package, show the output to see that the ``ncurses/system`` is skipped, and that consequently the ``Apt()`` or equivalent is not called.

Reply:
Simplified by the commit [95b4c57](https://github.com/conan-io/docs/pull/3980/commits/95b4c57e6d7733e5080df954f934d50736f4f457)